File: swagger/models/send_password_reset_email_request.py
from .base_model import BaseModel


class SendPasswordResetEmailRequest(BaseModel):
	"""
	Request to send a password reset email to a user

	Attributes:

		user_email
		The user’s email address. The software uses this parameter as the
		username.

		user_first_name
		The user’s first name. The software uses this parameter to verify
		the user.

		user_last_name
		The user’s last name. The software uses this parameter to verify the
		user.
	"""

	input_map = {
		'UserEmail': 'user_email',
		'UserFirstName': 'user_first_name',
		'UserLastName': 'user_last_name',
		}

	output_map = {
		'user_email': 'UserEmail',
		'user_first_name': 'UserFirstName',
		'user_last_name': 'UserLastName',
		}

	user_email = None
	user_first_name = None
	user_last_name = None



	def __init__(self, user_email, user_first_name, user_last_name):
		self.user_email = user_email
		self.user_first_name = user_first_name
		self.user_last_name = user_last_name

File: swagger/Models/SendPasswordResetEmailRequest.php
<?php

namespace App\MindBody\Models;

/**
 * Request to send a password reset email to a user
 */
class SendPasswordResetEmailRequest extends BaseModel
{
	/**
	 * The user’s email address. The software uses this parameter as the
	 * username.
	 * 
	 * @var string 
	 */
	public $userEmail;
	/**
	 * The user’s first name. The software uses this parameter to verify
	 * the user.
	 * 
	 * @var string 
	 */
	public $userFirstName;
	/**
	 * The user’s last name. The software uses this parameter to verify the
	 * user.
	 * 
	 * @var string 
	 */
	public $userLastName;
	/**
	 * Constructor
	 */
	public function __construct($userEmail, $userFirstName, $userLastName)
	{
		$this->userEmail = $userEmail;
		$this->userFirstName = $userFirstName;
		$this->userLastName = $userLastName;
	}

	protected function getInputMap()
	{
		return [
			'UserEmail' => ['userEmail', 'string', null],
			'UserFirstName' => ['userFirstName', 'string', null],
			'UserLastName' => ['userLastName', 'string', null],
			];
	}

	protected function getOutputMap()
	{
		return [
			'userEmail' => ['UserEmail', 'string', null],
			'userFirstName' => ['UserFirstName', 'string', null],
			'userLastName' => ['UserLastName', 'string', null],
			];
	}
}
